src/js/color_change.js

```diff
@@ -145,6 +142,21 @@ $(function() {
             courseId = lastAddedCourse[0] + 1;
         }
 
+        var courseSplit = course.split('-');
+        var courseCode = courseSplit[0].trim();

```

Comment:
We are already calling `trim` on `course` so we don't need it again.
```suggestion
        var courseCode = courseSplit[0];
```